.gitignore

```diff
@@ -0,0 +1,2 @@
+# Vim swap files
+*.sw?
```

Comment:
Editor-specific files should not be in gitignore: https://help.github.com/en/github/using-git/ignoring-files#excluding-local-files-without-creating-a-gitignore-file

Reply:
Agree.
Remove `.gitignore`